PROTOCOL.md

```diff
@@ -477,7 +477,7 @@ The following is an example `remove` action.
 ```
 
 ### Add CDC File
-The `cdc` action is used to add a [file](#change-data-files) containing only the data that was changed as part of the transaction. When change data readers encounter a `cdc` action in a particular Delta table version, they must read the changes made in that version exclusively using the `cdc` files. If a version has no `cdc` action, then the data in `add` and `remove` actions are read as inserted and deleted rows, respectively.
+The `cdc` action is used to add a file containing only the data that was changed as part of the transaction. The file might be a [Change Data File](#change-data-files) or a [Data File](#data-file) that does not contain any copied rows. When change data readers encounter a `cdc` action in a particular Delta table version, they must read the changes made in that version exclusively using the `cdc` files. If a version has no `cdc` action, then the data in `add` and `remove` actions are read as inserted and deleted rows, respectively.
```

Comment:
i still dont get it. is it your goal to say that the same parquet file can be put as both AddFile and AddCDCFile? if so, then this is not clear. 


Also if that is the goal, then its important also write down when a parquet data file can be considered as a CDC file? what constraints does that parquet file need to satisfy.

Reply:
Thanks @tdas. I reworded this section to make it more clear and clarify the requirements for the file added by the `AddCDCFile` action. 

In general the protocol does not refer to "Parquet files". It defines that:
- An `AddFile` action adds a `Data File` which is a file that is stored in the root directory.
- An `AddCDCFile` action adds a `Change Data File` which is a file stored in the `_change_data` directory, contains only data that was changes as part of a transaction (no copied rows) and has an extra `_change_type` column.

This is why I now define that:
"The `cdc` action is allowed to add a [Data File](#data-files) that is also added by an `add` action, when it does not contain any copied rows and the `_change_type` column is filled for all rows."